Sources/SwiftInspectorVisitors/StructVisitor.swift

```diff
@@ -150,11 +150,3 @@ public final class StructVisitor: SyntaxVisitor {
   private var structInfo: StructInfo?
   private var innerStructs = [StructInfo]()
 }
-
-public struct StructInfo: Codable, Equatable {
```

Comment:
this type is moved to above with no changes (unless you are in file-changed view, at which point the comment is moved to a new type)